src/webhooks/registry.ts

```diff
@@ -74,6 +74,20 @@ function isSuccess(result: any, deliveryMethod: DeliveryMethod, webhookId?: stri
             result.data.eventBridgeWebhookSubscriptionCreate.webhookSubscription,
         );
       }
+    case DeliveryMethod.PubSub:
+      if (webhookId) {
+        return Boolean(
+          result.data &&
+            result.data.pubSubWebhookSubscriptionUpdate &&
+            result.data.pubSubWebhookSubscriptionUpdate.webhookSubscription,
+        );
+      } else {
+        return Boolean(
+          result.data &&
+            result.data.pubSubWebhookSubscriptionCreate &&
+            result.data.pubSubWebhookSubscriptionCreate.webhookSubscription,
+        );
+      }
```

Comment:
Minor: not sure if this makes it easier to read, but we could pull-out the only difference in these blocks to de-duplicate some code:
```suggestion
      endpoint = !!webhookId ? "pubSubWebhookSubscriptionUpdate" : "pubSubWebhookSubscriptionCreate"
      return Boolean(
        result.data &&
          result.data[endpoint] &&
          result.data[endpoint].webhookSubscription,
      );
```

Or, if we really want to help out, something like:
```javascript
endpoint = ""
switch (deliveryMethod) {
  case DeliveryMethod.HTTP:
    endpoint = "webhookSubscription";
    break;
  case DeliveryMethod.EventBridge:
    endpoint = "eventBridgeWebhookSubscription";
    break;
  case DeliveryMethod.PubSub:
    endpoint = "pubSubWebhookSubscription";
    break;
  default:
    return false;
}
endpoint += !!webhookId ? "Update" : "Create";
return Boolean(
  result.data &&
    result.data[endpoint] &&
    result.data[endpoint].webhookSubscription,
);
```
Thoughts on this?

Reply:
Good idea, thanks!

Went with the second option, that eliminates a lot of repetition.